src/core/client_channel/subchannel_pool_interface.cc

```diff
@@ -55,8 +55,7 @@ std::string SubchannelKey::ToString() const {
   auto addr_uri = grpc_sockaddr_to_uri(&address_);
   return absl::StrCat(
       "{address=",
-      addr_uri.ok() ? addr_uri.value() : addr_uri.status().ToString(),
-      ", args=", args_.ToString(), "}");
+      addr_uri.ok() ? addr_uri.value() : addr_uri.status().ToString(), "}");
```

Comment:
The subchannel args are part of the key, so they're important to log here.  Please revert this change.